diff --git a/node_modules/react-native-thumbnail-video/src/components/thumbnail.js b/node_modules/react-native-thumbnail-video/src/components/thumbnail.js
index bc53a4f..2a388e1 100644
--- a/node_modules/react-native-thumbnail-video/src/components/thumbnail.js
+++ b/node_modules/react-native-thumbnail-video/src/components/thumbnail.js
@@ -61,15 +61,15 @@ export default class Thumbnail extends PureComponent {
     return null;
   }
 
-  UNSAFE_componentWillUpdate(nextProps) {
-    if (this.props.url === nextProps.url || !nextProps.url) {
-      return;
-    }
-
-    this.setState({
-      videoId: getVideoId(nextProps.url),
-    });
-  }
+  // UNSAFE_componentWillUpdate(nextProps) {
+  //   if (this.props.url === nextProps.url || !nextProps.url) {
+  //     return;
+  //   }
+
+  //   this.setState({
+  //     videoId: getVideoId(nextProps.url),
+  //   });
+  // }
 
   getType = () => TYPES[this.props.type];
 
